sci-libs/libticonv/libticonv-1.1.5.ebuild

```diff
@@ -21,18 +21,21 @@ DEPEND="${RDEPEND}
 
 DOCS=( AUTHORS LOGO NEWS README ChangeLog docs/api.txt )
 
+src_prepare() {
+	default
+	eautoreconf
+}
+
 src_configure() {
 	econf \
 		$(use_enable static-libs static) \
 		$(use_enable iconv)
 }
 
 src_install() {
-	default
 	if use doc; then
-		dohtml docs/html/*
-		docinto charsets
-		dohtml docs/charsets/*
+		HTML_DOCS=( docs/html/. )
+		dodoc -r docs/charsets
```

Comment:
Are you sure? I think if DOCS is unset einstalldocs searches for default files and I don't have to specify them manually. With dodoc I would just add additional docs.